lib/routes/account.js

```diff
@@ -332,6 +332,7 @@ module.exports = function (
             authPW: isA.string().min(64).max(64).regex(HEX_STRING).required(),
             // Obsolete contentToken param, here for backwards compat.
             contentToken: isA.string().optional(),
+            sendEmailIfUnverified: isA.boolean().optional(),
```

Comment:
I think it'd be worth a comment here to explain what the param is for, particularly if we plan to remove it at some future date.